BlinkID/lib/overlay_settings.dart

```diff
@@ -1,33 +1,39 @@
 import 'package:json_annotation/json_annotation.dart';
+
 export 'package:blinkid_flutter/overlays/blinkid_overlays.dart';
 
 part 'overlay_settings.g.dart';
 
-/// Base class for all overlay settings objects 
+/// Base class for all overlay settings objects
 @JsonSerializable()
 class OverlaySettings {
-    /// type of the overlay settings object
-    String? overlaySettingsType;
+  /// type of the overlay settings object
+  String? overlaySettingsType;
+
+  /// whether front camera should be used instead of the default camera
+  /// false by default
+  bool useFrontCamera = false;
 
-    /// whether front camera should be used instead of the default camera
-    /// false by default
-    bool useFrontCamera = false;
+  /// whether beep sound will be played on successful scan
+  /// false by default
+  bool enableBeep = false;
 
-    /// whether beep sound will be played on successful scan
-    /// false by default
-    bool enableBeep = false;
+  /// Flip the camera view horizontally, only when [useFrontCamera] is true.
+  ///
+  /// This only applies when the current platform is Android.
```

Comment:
Why Android only?

Reply:
Since we only face missing issues on Android. On iOS the camera view will always be mirrored. But on Android, manufacturers may choose to change how this is handled hence we will have this option to flip the view on some devices.